### PR TITLE
kgoldrunner: init at 21.08.0

### PR DESCRIPTION
**Pull request by alyssais**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 138923` at 2256a46d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/a2413dde453c79d3beb85b2c54f7e7de">libsForQt512.kgoldrunner</a></li>
  </ul>
</details>
<details>
  <summary>1 package skipped due to time constraints:</summary>
  <ul>
    <li>libsForQt514.kgoldrunner</li>
  </ul>
</details>
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>libsForQt5.kgoldrunner (libsForQt515.kgoldrunner ,plasma5Packages.kgoldrunner)</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Comment by SuperSandro2000** [comment]
Please set it broken for qt 5.12.

**Comment by papojari** [comment]
@alyssais pls update your fork to fix merge conflict.

File: pkgs/applications/kde/kgoldrunner.nix [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  pname = "kgoldrunner";
+
+  nativeBuildInputs = [ extra-cmake-modules ];
+  buildInputs = [ kio kxmlgui libkdegames ];
+
+  meta = with lib; {
+    homepage = "https://apps.kde.org/kgoldrunner/";
+    description = "Hunt gold, dodge enemies and solve puzzles";
+    license = licenses.gpl2Plus;
+    platforms = platforms.linux;
```
Comment (papojari) [inline pkgs/applications/kde/kgoldrunner.nix:13]:
```suggestion
    platforms = platforms.linux;
    broken = versionOlder qtbase.version "5.13";
```
Like that @SuperSandro2000? Or is it just broken for 5.12?

**Comment by gepbird** [comment]
We have already `kdePackages.kgoldrunner`

**Pull request closed without merging**